docs/source/tutorials/token_classification.ipynb

```diff
@@ -173,51 +178,56 @@
-    "    filepaths = ['data/train.txt', 'data/valid.txt', 'data/test.txt'] \n",
-    "    given_entities = ['O', 'B-MISC', 'I-MISC', 'B-PER', 'I-PER', 'B-ORG', 'I-ORG', 'B-LOC', 'I-LOC']\n",
-    "    merged_entities = ['O', 'MISC', 'PER', 'ORG', 'LOC'] \n",
-    "    entity_map = {entity: i for i, entity in enumerate(given_entities)} \n",
+    "```ipython3\n",
+    "# Note: This pulldown content is for docs.cleanlab.ai, if running on local Jupyter or Colab, please ignore it.\n",
+    "\n",
+    "\n",
+    "given_entities = ['O', 'B-MISC', 'I-MISC', 'B-PER', 'I-PER', 'B-ORG', 'I-ORG', 'B-LOC', 'I-LOC']\n",
+    "merged_entities = ['O', 'MISC', 'PER', 'ORG', 'LOC'] \n",
+    "entity_map = {entity: i for i, entity in enumerate(given_entities)} \n",
     "\n",
-    "    def readfile(filepath, sep=' '): \n",
-    "        lines = open(filepath)\n",
+    "def readfile(filepath, sep=' '): \n",
+    "    lines = open(filepath)\n",
     "\n",
-    "        data, sentence, label = [], [], []\n",
-    "        for line in lines:\n",
-    "            if len(line) == 0 or line.startswith('-DOCSTART') or line[0] == '\\n':\n",
-    "                if len(sentence) > 0:\n",
-    "                    data.append((sentence, label))\n",
-    "                    sentence, label = [], []\n",
-    "                continue\n",
-    "            splits = line.split(sep) \n",
-    "            word = splits[0]\n",
-    "            if len(word) > 0 and word[0].isalpha() and word.isupper():\n",
-    "                word = word[0] + word[1:].lower()\n",
-    "            sentence.append(word)\n",
-    "            label.append(entity_map[splits[-1][:-1]])\n",
+    "    data, sentence, label = [], [], []\n",
+    "    for line in lines:\n",
+    "        if len(line) == 0 or line.startswith('-DOCSTART') or line[0] == '\\n':\n",
+    "            if len(sentence) > 0:\n",
+    "                data.append((sentence, label))\n",
+    "                sentence, label = [], []\n",
+    "            continue\n",
+    "        splits = line.split(sep) \n",
+    "        word = splits[0]\n",
+    "        if len(word) > 0 and word[0].isalpha() and word.isupper():\n",
+    "            word = word[0] + word[1:].lower()\n",
+    "        sentence.append(word)\n",
+    "        label.append(entity_map[splits[-1][:-1]])\n",
     "\n",
-    "        if len(sentence) > 0:\n",
-    "            data.append((sentence, label))\n",
+    "    if len(sentence) > 0:\n",
+    "        data.append((sentence, label))\n",
     "\n",
-    "        given_words = [d[0] for d in data] \n",
-    "        given_labels = [d[1] for d in data] \n",
+    "    given_words = [d[0] for d in data] \n",
+    "    given_labels = [d[1] for d in data] \n",
     "\n",
-    "        return given_words, given_labels \n",
+    "    return given_words, given_labels \n",
     "\n",
-    "    given_words, given_labels = [], [] \n",
+    "filepaths = ['data/train.txt', 'data/valid.txt', 'data/test.txt'] \n",
```

Comment:
## Markdown code snippet

Moved initialization of `filepaths` closer to where it is used.

This way:
- The code above can be hidden in the docs
- The subsequent code snippet is "self-contained" and can be shown in the docs.